Source/UIKit/UITextView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UITextView.swift
+//  Rex
+//
+//  Created by Rui Peres on 05/04/2016.
+//  Copyright © 2016 Neil Pankey. All rights reserved.
+//
+
+import ReactiveCocoa
+import UIKit
+import enum Result.NoError
+
+extension UITextView {
+    
+    /// Sends the textView's string value whenever it changes.
+    public var rex_textSignal: SignalProducer<String, NoError> {
+        return NSNotificationCenter.defaultCenter()
+            .rac_notifications(UITextViewTextDidChangeNotification, object: self)
+            .filterMap { notification in
```

Comment:
Sorry, yes — that was just me not paying enough attention.
`($0.notification.object as? UITextView)?.text`